src/test/java/teammates/test/pageobjects/InstructorStudentListPage.java

```diff
@@ -51,13 +51,15 @@ public boolean isCopyEmailButtonVisible() {
         return copyEmailButton.isDisplayed();
     }
     
-    public boolean isCopyEmailPopoverVisible() {
+    public void waitForCopyEmailPopoverVisible() {
         String selector = "#copy-email-button + div.popover";
-        return isElementVisible(By.cssSelector(selector));
+        waitForElementPresence(By.cssSelector(selector));
```

Comment:
Let's use `waitForElementVisibility` instead as `waitForElementPresence` only checks if element is present is in DOM, regardless of whether it is visible or not.